assets/javascripts/discourse/components/group-reports-nav-item.js.es6

```diff
@@ -0,0 +1,34 @@
+import { ajax } from "discourse/lib/ajax";
+
+export default Ember.Component.extend({
+  hasAssignments: false,
+
+  checkAssignments() {
+    ajax(`/assign/assigned/${this.group.name}.json`).then(response => {
+      let render = this.currentUser.admin;
+      if (
+        (this.currentUser.hasOwnProperty("groups") &&
+          this.currentUser.groups !== "undefined") ||
+        !render
+      ) {
+        this.currentUser.groups.forEach(element => {
+          if (element.name === this.attrs.group.value.name) {
+            render = true;
+            return false;
+          }
+        });
+      }
+      this.set(
+        "hasAssignments",
+        render &&
+          this.siteSettings.assign_enabled &&
+          response.topic_list.topics.length > 0
+      );
+    });
```

Comment:
We don't really want to be making ajax requests while rendering components. Let's simplify this to match [the per-user behaviour](https://github.com/discourse/discourse-assign/blob/f312ece4a97a7bf40c1d3a734b754df1291758dc/assets/javascripts/discourse-assign/connectors/user-activity-bottom/assigned-list.hbs#L1).

If `current_user.can_assign` is true, show the tab. If not, then hide it.

Reply:
Sure, it's now done as specified in the new commit😀